src/applications/vaos/tests/new-appointment/components/VAFacilityPage/VAFacilityPageV2.multi.unit.spec.jsx

```diff
@@ -456,7 +456,7 @@ describe('VAOS integration: VA flat facility page - multiple facilities', () =>
     ).to.exist;
   });
 
-  it('should show not supported message when direct is supported and not eligible, and requests are not supported', async () => {
+  it.skip('should show not supported message when direct is supported and not eligible, and requests are not supported', async () => {
```

Comment:
Is this test not working?

Reply:
Oh no I missed it and so did the linter 🤔

Reply:
Fixed